os_migrate/roles/export_workloads/tasks/main.yml

```diff
@@ -32,6 +32,7 @@
     name: "{{ item['id'] }}"
     migration_params:
       boot_disk_copy: "{{ os_migrate_workloads_boot_disk_copy }}"
+      data_copy: "{{ os_migrate_workloads_data_copy }}"
```

Comment:
Ah yes that's a good solution to allow us to override the default for all exports, i actually forgot we had this for boot_disk_copy.